packages/api/src/common/router/index.d.ts

```diff
@@ -11,6 +11,11 @@ export interface Cookie {
 
 export interface Request {
   invalidateSession(): void;
+  /**
+   * Synchronizes local session state with the shared app session state.
+   * @since Sitevision 2023.03.1
+   */
+  updateSession(): void;
```

Comment:
This returns a session, as of now typed as any.
```suggestion
  updateSession(): any;
```